pkg/registry/core/pod/storage/eviction.go

```diff
@@ -303,11 +303,11 @@ func resourceVersionIsUnset(options *metav1.DeleteOptions) bool {
 }
 
 func createTooManyRequestsError(name string) error {
-	// TODO(mml): Add a Retry-After header.  Once there are time-based
+	// TODO(mml): Once there are time-based
```

Comment:
suggest switching to an active user

Reply:
Without a clear candidate, is it better to remove the username, or leave it as-is?